app/uploaders/image_uploader.rb

```diff
@@ -9,7 +9,7 @@ class ImageUploader < FileUploader
     process resize_to_fit: CONFIG['image_embed_size'] || [800, 600]
   end
 
-  def default_url
+  def default_url(*args)
```

Comment:
If we don't need `*args`, please remove it.

Reply:
Done